src/mapInteractor.js

```diff
@@ -192,11 +203,35 @@ var mapInteractor = function (args) {
          * (which would propably require detecting keyup events). */
         focusHighlight: true
       },
-
+      /* Set alwaysTouch to false to only add touch support on devices that
+       * report touch support.  Set to true to add touch support on all
+       * devices. */
+      alwaysTouch: false,
       wheelScaleX: 1,
       wheelScaleY: 1,
       zoomScale: 1,
       rotateWheelScale: 6 * Math.PI / 180,
+      /* The minimum angle of rotation (in radians) before the
+       * geo_action.zoomrotate action will allow rotation.  Set to 0 to always
+       * include rotation. */
+      zoomrotateMinimumRotation: 5.0 * Math.PI / 180,
+      /* The minimum angle of rotation (in radians) before the
+       * geo_action.zoomrotate action will reverse the rotation direction.
+       * This helps reduce chatter when zooms and pans are combined with
+       * rotations. */
+      zoomrotateReverseRotation: 4.0 * Math.PI / 180,
```

Comment:
thanks for the info. I will try playing with some other numbers if i get a chance.